pypesto/profile/profile.py

```diff
@@ -24,7 +24,7 @@ def parameter_profile(
     profile_index: Iterable[int] = None,
     profile_list: int = None,
     result_index: int = 0,
-    next_guess_method: Union[Callable, str] = "adaptive_step_regression",
+    next_guess_method: Union[Callable, str] = "adaptive_step_order_1",
```

Comment:
Hmm not sure. In all testing I did, it did perform worse than order 1. But it might be still that I just didn't find an example in which it just might be better. So I wouldn't remove it just to be able to test further, as I will be working on some further improvements of profiling soon :)